modules/openapi-generator/src/main/resources/python-pydantic-v1/api_test.mustache

```diff
@@ -15,14 +15,14 @@ class {{#operations}}Test{{classname}}(unittest.{{#asyncio}}IsolatedAsyncio{{/as
         self.api = {{classname}}()
 
     async def asyncTearDown(self) -> None:
-        pass
+        await self.api.api_client.close()
```

Comment:
can you please review the build errors after updating the test files manually?

https://github.com/OpenAPITools/openapi-generator/actions/runs/12608561178/job/35141150586?pr=20400

Reply:
This is because the synchronous API client does not have this `close` method. I checked the code a bit and found that currently only the client in `python-pydantic-v1` has this. The `close` method was removed in https://github.com/OpenAPITools/openapi-generator/pull/16802/commits/4d3d312aee20229aa3cbf7406ec722843d8d6f8a. The author reverted some changes in #16979 but not the pool thing. To fix this for now, I'm just reverting the change to the API client in `python`.

Reply:
Yes, the sync client does not need to perform a `close`

Reply:
thanks for the info

cc @fa0311 whos' the author of https://github.com/OpenAPITools/openapi-generator/commit/4d3d312aee20229aa3cbf7406ec722843d8d6f8a

Reply:
👌